demo/app/lib/views/credential_preview.dart

```diff
@@ -158,6 +161,12 @@ class CredentialPreviewState extends State<CredentialPreview> {
                       ),
                       PrimaryButton(
                           onPressed: () async {
+                            var ackResp = await WalletSDKPlugin.requireAcknowledgment();
+                            if (ackResp == true) {
+                              await WalletSDKPlugin.acknowledgeSuccess();
+                            } else {
+                              await WalletSDKPlugin.acknowledgeReject();
```

Comment:
The acknowledgeReject call should be triggered for the `Cancel` button - please remove the else condition here.

Add following condition for `Cancel` button event.

```dart
var ackResp = await WalletSDKPlugin.requireAcknowledgment();
if (ackResp == true) {
  await WalletSDKPlugin.acknowledgeReject();
}
```